neard/Cargo.toml

```
@@ -13,7 +13,8 @@ path = "src/main.rs"
 name = "neard"
 
 [dependencies]
-clap = "=3.0.0-beta.2"
+clap = { version = "3.0.0-rc.7", features = ["cargo"] }
+clap_derive = "3.0.0-rc.7"
```

```suggestion
clap = { version = "3.0.0-rc.7", features = ["cargo", "derive"] }
```

I believe that's the intended idiomatic way of using clap. For something neare the bottom of dep graph, like `serde`, depdending on runtime crate and derive crate separately *might* help with compile times, but that's not the case for clap.

---

@matklad Thanks, I updated the code.